artman/pipelines/grpc_generation.py

```diff
@@ -118,7 +118,7 @@ def get_grpc_codegen_tasks(self, **kwargs):
             protoc_tasks.ProtoDescGenTask,
             protoc_tasks.ProtoAndGrpcCodeGenTask,
             package_metadata_tasks.PackageMetadataConfigGenTask,
-            package_metadata_tasks.PythonProtoPackageMetadataGenTask,
```

Comment:
I will do a manual test to make sure it works for Python but I feel this maybe the case that the Python unit tests should catch.